### ISSUE TITLE
Found a crash with `--header` rule

### ISSUE THREAD

**Issue by honghaoz** [nicklockwood/SwiftFormat#774]
Just found a crash on the `--header` rule:

### Test files
*.swiftformat*:
```
--header "\nCopyright © {year} Foo. All rights reserved." 

```

*test.swift*:
```swift
// Copyright © 2020 Example. All rights reserved.

```

### Expected result:
The *test.swift* with content
```swift
// Copyright © 2020 Foo. All rights reserved.

```

### Current result
```
➜  testSwiftFormat swiftformat . --verbose
Running SwiftFormat...
Reading config file at /Users/username/Desktop/testSwiftFormat/.swiftformat
Formatting /Users/username/Desktop/testSwiftFormat/test.swift
[1]    11146 illegal hardware instruction  swiftformat . --verbose
```
*test.swift* doesn't change.

If update the *.swiftformat* to (notice the beginning `\n`):
```
--header "Copyright © {year} Foo. All rights reserved." 

```

The result file is:
```swift
// Copyright © 2020 Foo. All rights reserved.

// Copyright © 2020 Example. All rights reserved.

```

### Context
Current version: 0.47.0

### Sample
[testSwiftFormat.zip](https://github.com/nicklockwood/SwiftFormat/files/5397307/testSwiftFormat.zip)

**Comment by honghaoz** [comment]
1) With rule:
```
--header "Copyright © {year} Foo. All rights reserved.\n" 

```
or
```
--header "Hey\n" 

```
or
```
--header "\nHey" 

```
The same crash.

2) With rule:
```
--header "Copyright © {year} Foo. All rights reserved.\n\n" 

```
result is:
```swift
// Copyright © 2020 Foo. All rights reserved.
//
//

// Copyright © 2020 Example. All rights reserved.

```

3) With rule:
```
--header "\n\nCopyright © {year} Foo. All rights reserved." 

```
result is:
```swift
//
//
// Copyright © 2020 Foo. All rights reserved.

// Copyright © 2020 Example. All rights reserved.

```

4) With rule:
```
--header "\nHey\n" 

```
result is:
```
//
// Hey
//

// Copyright © 2020 Example. All rights reserved.

```

**Comment by nicklockwood** [comment]
Fixed in 0.47.1
